src/backend/src/services/change-requests.services.ts

```diff
@@ -366,21 +366,26 @@ export default class ChangeRequestsService {
           include: {
             workPackage: {
               include: {
-                project: { include: { team: true, wbsElement: true } }
+                project: { include: { teams: true, wbsElement: true } }
               }
             }
           }
         }
       }
     });
 
-    const team = createdCR.wbsElement.workPackage?.project.team;
-    if (team) {
+    const teams = createdCR.wbsElement.workPackage?.project.teams;
+
+    if (!teams) {
+      throw new HttpException(400, 'This project needs to be assigned to a team to create a task!');
```

Comment:
this is for creating an activation CR, not for creating a task. I think we should j do it how it was before, only send the slack notifs if we have teams